htdocs/accountancy/report/result.php

```diff
@@ -35,12 +36,14 @@
 $langs->load("compta");
 
 $mesg = '';
-$action = GETPOST('action','aZ09');
+$action = GETPOST('action');
```

Comment:
Why removing 'aZ09'
For $action var, the GETPOSTmust be GETPOST('action','aZ09') everywhere. If should works for you two. If not, just tell me what is value of action you use. There is probably a bug somewhere else...